css_tasks/task6/index.html

```diff
@@ -0,0 +1,297 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Task6</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Inconsolata|Playfair+Display|Ubuntu:300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <div class="container">
+      <!-- HEADER -->
+      <header class="header">
+        <span class="header__logo">minimø</span>
+        <nav class="header__navigation">
+          <ul class="header__navigation-list">
+            <li class="header__navigation-list--item">Lifestyle</li>
+            <li class="header__navigation-list--item">Photodiary</li>
+            <li class="header__navigation-list--item">Music</li>
+            <li class="header__navigation-list--item">Travel</li>
+          </ul>
+        </nav>
+        <img src="assets/svg/bars-solid.svg" alt="Mobile menu icon" class="header__mobile-icon">
+      </header>
+      <!-- JUMBOTRON -->
+      <div class="jumbotron">
+        <picture>
+          <source
+            srcset="assets/image@3x-min.jpg"
+            media="(min-width: 1200px)"
+          />
+          <source srcset="assets/image@2x-min.jpg" media="(min-width: 768px)" />
+          <source srcset="assets/image-min.jpg" media="(max-width: 767.98px)" />
+          <img
+            srcset="assets/image-min.jpg"
```

Comment:
`src` attribute is [required](https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-src)

Reply:
Done